apis/r/src/Makevars.in

```diff
@@ -17,7 +18,13 @@ all: $(SHLIB)
 	if [ `uname -s` = 'Darwin' ] && [ -f tiledbsoma.so ]; then \
 	    if [ -f ../inst/tiledb/lib/libtiledb.dylib ] ; then \
 	        install_name_tool -change libz.1.dylib @rpath/libz.1.dylib ../inst/tiledb/lib/libtiledb.dylib; \
-		install_name_tool -add_rpath @loader_path/../tiledb/lib tiledbsoma.so; \
-	    	install_name_tool -add_rpath @loader_path/../tiledbsoma/lib tiledbsoma.so; \
+	        install_name_tool -add_rpath @loader_path/../tiledb/lib tiledbsoma.so; \
+	        install_name_tool -add_rpath @loader_path/../tiledbsoma/lib tiledbsoma.so; \
+	    fi; \
+	    if [ -f /usr/local/lib/libtiledb.dylib -o -f /usr/local/lib/libtiledbsoma.dylib ] ; then \
+	        install_name_tool -add_rpath /usr/local/lib tiledbsoma.so; \
 	    fi; \
```

Comment:
That should definitely help and repair something I broke for the 'local dev, on a mac' case.  I cannot quite figure out what / if anything we can / should do if one of the two is present. 

So maybe even test with a `-a` instead of `-o` ?

Reply:
@eddelbuettel I think `-o` since we are _adding_ the rpath.

Reply:
Yes but as I wrote: what if we satisfy the entry condition of 

     if [ -f /usr/local/lib/libtiledb.dylib -o -f /usr/local/lib/libtiledbsoma.dylib ]
     
if _one but not the other_ library is present?

Reply:
@eddelbuettel that is precisely what I was referring to when I wrote

>  I'll test on my Mac with both/one/neither installed

Reply:
Ok -- good enough.  Let's move forward with this as it does improve on the status quo.  If we need further fixes we will make them.